src/libpmemobj/heap.c

```diff
@@ -913,6 +913,16 @@ heap_end(struct palloc_heap *h)
 	return &last_zone->chunks[last_zone->header.size_idx];
 }
 
+/*
+ * heap_rt_get_narenas -- returns the number of arenas from rt heap
+ */
+unsigned
+heap_rt_get_narenas(struct palloc_heap *heap)
```

Comment:
rename `heap_get_narenas` to `heap_get_procs`, and this function to `heap_get_narenas`.